src/test/relays/TrustedRelay.t.sol

```diff
@@ -713,7 +779,7 @@ contract TrustedRelayTest is DSTest {
 
         (uint8 v, bytes32 r, bytes32 s) = hevm.sign(sk, signHash);
 
-        relay.relay(
+        assertTrue(!_tryRelay(
```

Comment:
I see this change in the approach of the test check, but it wasn't change in other ones.

Reply:
I had introduced `_tryRelay` here to get around a "stack too deep" issue but I've now applied it to the rest of the `relay()` calls for consistency.

Reply:
Yeah that's perfect, but what I was actually referring here is that in this specific test we removed the `Fail` from the test title and we are checking `assertTrue(! ...)`. So this is a better approach as at least we can prove that the test is failing in the `relay` call (middle ground solution regarding the best one which is using forge and checking revert event).
So, I'm saying we should do it everywhere where we have today a `testFail`.

Reply:
(also for standardization between tests)

Reply:
Agreed and changed.